src/components/Breadcrumbs/Breadcrumbs.scss

```diff
@@ -30,16 +30,16 @@ $stacking-order: (
   margin:
     (-0.5 * $difference-between-touch-area-and-backdrop) (-1 * $horizontal-padding) (-0.5 * $difference-between-touch-area-and-backdrop) (-0.5 * $horizontal-padding);
   padding: $vertical-padding $horizontal-padding;
-  color: color('ink', 'lighter');
+  color: var(--p-text-subdued-on-surface, color('ink', 'lighter'));
   text-decoration: none;
 
   &:hover,
   &:active {
-    color: color('ink');
+    color: var(--p-text-on-surface, color('ink'));
     text-decoration: none;
 
     .Icon {
-      @include recolor-icon(color('ink'));
+      @include recolor-icon(var(--p-icon-on-surface, color('ink')));
```

Comment:
I've used `--p-icon` but this causes an issue in dark mode where the icon doesn't match the text color but it does on light mode. What's the approach to fix this inconsistent:

1. change `iconSubduedOnSurface` to match`textSubduedOnSurface` in `color-adjustments.ts`?
2. or just use the `text[Subdued]OnSurface` for both the icon and the text?

Reply:
This is fine, we can adjust the colors later.